pkg/cliClient/cliClient_test.go

```diff
@@ -174,6 +199,30 @@ func TestGetVersionMessage(t *testing.T) {
 	}
 }
 
+func TestPublishPolicies(t *testing.T) {
+	tests := []*PublishPoliciesTestCase{
+		test_publishPolicies_success(),
```

Comment:
add a failed test. we need to make sure we can handle errors from the backend, this should happen a lot